schema.sql

```diff
@@ -1,12 +1,27 @@
 /* Database schema to keep the structure of entire database. */
 CREATE database vet_clinic;
 
+CREATE TABLE owners (
+    id INT GENERATED ALWAYS AS IDENTITY NOT NULL,
+    full_name VARCHAR(250),
+    age INT,
+    PRIMARY KEY(id)
+);
+
+CREATE TABLE species (
+    id INT GENERATED ALWAYS AS IDENTITY NOT NULL,
+    name VARCHAR(250),
+    PRIMARY KEY(id)
+);
+
 CREATE TABLE animals (
-    id INT,
+    id INT GENERATED ALWAYS AS IDENTITY NOT NULL,
     name VARCHAR(100),
     date_of_birth DATE,
     escape_attempts INT,
     neutered BOOLEAN,
     weight_kg FLOAT(2),
-    species VARCHAR(250)
-);
+    species_id INT REFERENCES species(id),
+    owner_id INT REFERENCES owners(id),
+    PRIMARY KEY(id) 
+);
```

Comment:
- [ ] Please as requested by the previous code reviewer, From the project requirement, you should modify the `animals` table: :pray: 
  - Remove column `species`
  - Add column `species_id` which is a foreign key referencing the `species` table
  - Add column `owner_id` which is a foreign key referencing the `owners` table
  
**Make sure that you add your SQL that to perform these tasks mentioned above to this `schema.sql`**

Reply:
Still don't understand what you're saying. I don't have the column `species` and I have the columns `species_id` and `owner_id` in the animals table.

CREATE TABLE animals (
    id INT GENERATED ALWAYS AS IDENTITY NOT NULL,
    name VARCHAR(100),
    date_of_birth DATE,
    escape_attempts INT,
    neutered BOOLEAN,
    weight_kg FLOAT(2),
    species_id INT REFERENCES species(id),
    owner_id INT REFERENCES owners(id),
    PRIMARY KEY(id) 
);